tests/mmh3/bif.zeek

```diff
@@ -1,27 +1,19 @@
 # @TEST-EXEC: zeek -b Zeek::MoreHashes %INPUT >out
 # @TEST-EXEC: btest-diff out
 
-function my_assert(r: bool, msg: string) {
-	if ( r )
-		return;
-
-	print msg;
-	exit(1);
-}
-
 event zeek_init() &priority=0 {
 	local h = mmh3_hash_init();
 	mmh3_hash_update(h, "zeek");
 	local r = mmh3_hash_finish(h);
-	my_assert(r == 1898642774, fmt("expected 1898642774 got %s", r));
+	assert r == 1898642774, fmt("expected 1898642774 got %s", r);
```

Comment:
As an aside, this is IMO a completely reasonable, but it could break forks enabling ZAM since ZAM unconditionally removes `assert`s so it would be completely unusable for testing. Not sure why we do that, and also not really our problem here.

Reply:
The will know to set `ZEEK_ZAM_KEEP_ASSERTS=1` :-)